functions/Invoke-DbaLogShipping.ps1

```diff
@@ -1092,13 +1093,12 @@ The script will show a message that the copy destination has not been supplied a
 				}
 				Write-Message -Message "Backup network path set to $DatabaseBackupNetworkPath." -Level Verbose
 
-
 				# Checking if the database network path exists
-				if (-not (Test-Path -Path $DatabaseBackupNetworkPath -Credential $SourceCredential)) {
+				if ((Test-DbaSqlPath -Path $DatabaseBackupNetworkPath -SqlInstance $SourceSqlInstance -SqlCredential $SourceCredential) -ne $true) {
 					# To to create the backup directory for the database 
 					try {
 						Write-Message -Message "Backup network path not found. Trying to create it.." -Level Verbose
-						New-Item $DatabaseBackupNetworkPath -ItemType Directory -Credential $SourceCredential | Out-Null
+                        New-DbaSqlDirectory -Path $DatabaseBackupNetworkPath -SqlInstance $SourceSqlInstance -SqlCredential $SourceCredential
```

Comment:
While this may work in most cases `New-DbaSqlDirectory` utilizes `xp_create_subdir`. It again utilizes permissions based on SQL Server which may not have rights to the directory, while the `$SourceCredential` will. Although the reverse can also be the same as the `$SourceCredential` may  not have rights to connect to the given SQL Server instance. 

I would say the safer process would be to leave the use of `New-Item` and just wrap it in the `Invoke-Command2` call.

A variable could be added in the begin block to determine if a remote connection has to be made but in generally we just go ahead and assume it will be.